src/rgw/rgw_perf_counters.cc

```diff
@@ -90,6 +91,15 @@ void add_rgw_op_counters(PerfCountersBuilder *lpcb) {
   lpcb->add_time_avg(l_rgw_op_list_buckets_lat, "list_buckets_lat", "List buckets latency");
 }
 
+void add_rgw_topic_counters(PerfCountersBuilder *lpcb) {
+  // description must match general rgw counters description above
```

Comment:
it looks like this comment was copy/pasted from above, and doesn't make sense to me here